### ISSUE TITLE
Numerical issue with Beta Binomial logccdf

### ISSUE THREAD

**Issue by MFairley** [JuliaStats/Distributions.jl#1177]
I'm having a numerical issue evaluating the logccdf of the Beta Binomial. Here's an example.

```
d = BetaBinomial(200, 136, 8466)
logccdf(d, 10)
```

```
ERROR: DomainError with -9.256893004350175e-5:
log will only return a complex result if called with a complex argument. Try log(Complex(x)).
Stacktrace:
 [1] throw_complex_domainerror(::Symbol, ::Float64) at ./math.jl:33
 [2] log(::Float64) at ./special/log.jl:285
 [3] logccdf(::BetaBinomial{Float64}, ::Int64) at /Users/mfairley/.julia/packages/Distributions/jFoHB/src/univariates.jl:420
 [4] top-level scope at REPL[4]:1
```

**Comment by devmotion** [comment]
I can reproduce this issue on master but not with https://github.com/JuliaStats/Distributions.jl/pull/1171. It seems the change to `pdf(d::BetaBinomial, x) = exp(logpdf(d, x))` fixes this specific numerical issue.

**Comment by andreasnoack** [comment]
Closed by #1171
